lib/model/migrations/20220830-01-create-dataset_def.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = async (db) => {
+  await db.schema.createTable('dataset_defs', (t) => {
```

Comment:
Nit-picking the `dataset_defs` table name... I guess if there's no definition stored in that table, it could have a different name? `dataset_sources`? `dataset_form_sources`? `dataset_form_defs`? I'm not sure.

The closest example I can think of in the existing code is the `submission_attachments` table that stores the `blobId` <---> `submissionDefId` relation with some other stuff. Well, that stuff like attachment `name` is pretty important, maybe this is a bad example. the `assignments` table is also pretty minimal and just captures links between things.

If you can't think of a better name, the current table name is fine.

Reply:
Without looking at code, I would expect `dataset_defs` do define the dataset's schema.

Reply:
renamed the table to `dataset_form_defs`